src/app/print_blockchain_snark_vk/print_blockchain_snark_vk.ml

```diff
@@ -3,20 +3,31 @@ open Core_kernel
 let () =
   Async.Thread_safe.block_on_async_exn (fun () ->
       let () = Format.eprintf "Generating transaction snark circuit..@." in
+      let network_constants =
+        Option.value_map ~default:Runtime_config.Network_constants.dev
```

Comment:
This executable is actually run as part of the build, so you are forced to supply a default value here